server/src/main/java/org/apache/kafka/server/config/KafkaConfig.java

```diff
@@ -0,0 +1,146 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kafka.server.config;
+
+import org.apache.kafka.common.config.TopicConfig;
+import static org.apache.kafka.server.config.ServerTopicConfigSynonyms.LOG_PREFIX;
+
+public class KafkaConfig {
+
+    /** ********* Log Configuration ***********/
+    public final static String NUM_PARTITIONS_PROP = "num.partitions";
```

Comment:
> For consistency with the other *Configs classes, should we used the CONFIG suffix instead of PROP?
I thought about it but I wan't sure which would make it easier for people to review. 
We also might need to do the same for LogCleaner configs as well. 

> Would it also make sense to colocate the CONFIG and DOC like we do in the other Config classes?
I considered this but I found it tricky for some docs that refer to more than 1 config in the string interpolation.  For example the `LOG_DIR_DOC` and `LOG_DIRS_DOC` refer to both `LOG_DIR_PROP` and `LOG_DIRS_PROP` same goes to `LOG_ROLL_TIME_HOURS_DOC` and `LOG_ROLL_TIME_MILLIS_DOC`. I thought this why they are grouped at the end after defining the properties names first.

Reply:
I just pushed a commit to replace `PROP` with `CONFIG` suffix.